projecttacoma/measure-repository
# Publishable write

# Summary
Implement/ensure publishable artifact repository minimum write capabilities (Publish, Retire, Archive) as defined in the CRMI IG [here](https://hl7.org/fhir/uv/crmi/1.0.0-snapshot/artifact-repository-service.html#publishable-artifact-repository).

## New behavior

## Code changes

# Testing guidance
- `npm run check`
- Set the AUTHORING environment variable to false in your local .env file to test publishable-limited capabilities.
- `npm run start:service`
- Run db:postBundle, example `npm run db:postBundle {bundle path}`
  - Default assumes that the server is running on http://localhost:3000/4_0_1, or you can specify a server path as an additional argument on the end
- Test out the capabilities and limitations listed in the README for the publishable repository server (and authoring if it makes sense) using Insomnia
  - Can see added jest tests for more specific examples of things to test
